script.js

```diff
@@ -0,0 +1,31 @@
+const menu = document.getElementById('myTopnav');
+const navItems = document.getElementById('navItems').getElementsByTagName('a');
+const navItem = document.getElementById('navItem').getElementsByTagName('h3');
+
+function mobileMenu() {
+  if (menu.className === 'nav-items') {
+    menu.className += ' responsive';
+  } else {
+    menu.className = 'nav-items';
+  }
+}
+
+document.getElementById('menuIcon').addEventListener('click', mobileMenu);
+
+function showNavItems() {
+  for (let i = 0; i < navItems.length; i += 1) {
+    navItems[i].style.opacity = '1';
+    navItems[i].style.transform = 'translateY(0)';
+  }
+}
+
+window.addEventListener('load', showNavItems);
+
+function showNavItem() {
```

Comment:
Please what does this function do?

Reply:
It was for the mobile responsiveness but i've updated the function now to a different one